tests/regression/package/test_package_regression.py

```diff
@@ -23,6 +23,7 @@ def tearDown(self):
     @parameterized.expand(
         [
             "aws-serverless-api.yaml",
+            "aws-serverless-httpapi.yaml",
```

Comment:
Not sure we want to add it here. These tests are to test regressions against AWS CLI. I don't think we need this covered for this addition.